src/pure.js

```diff
@@ -149,7 +149,7 @@ function renderRoot(
         return template.content
       }
     },
-    ...getQueriesForElement(baseElement, queries),
```

Comment:
Could you check what other testing libraries do? This might've been intended because of portals. But the lack of tests is suspicious.